validation/OpenSpaceToolkit/Astrodynamics/data/test/test_parser_create_satellite_system.yaml

```diff
@@ -0,0 +1,19 @@
+# Test create satellite system function
+success-no-prop-system:
+  data:
+    spacecraft:
+      mass: 100.0
+      drag-cross-section: 1.0
+      drag-coefficient: 2.2
+
```

Comment:
(very nit) Some yaml have extra lines in between test cases when others don't. No issues when parsing?

Reply:
Nope, no issues when parsing because its just searching for the keys when parsing. But I will adjust this for consistency